src/content/docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial.mdx

```diff
@@ -16,7 +16,7 @@ To use NerdGraph, you'll need a [user key](/docs/apis/nerdgraph/get-started/intr
 
 ## Overview of tutorial [#overview]
 
-With New Relic’s [applied intelligence](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-applied-intelligence/), you can create custom [decisions](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions) that govern how your incidents are correlated. One kind of custom decision logic uses the concept of "topology," which is a representation of your service map: how the services and resources in your infrastructure relate to one another.
+With New Relic’s [alerts](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-applied-intelligence/), you can create custom [decisions](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions) that govern how your incidents are correlated. One kind of custom decision logic uses the concept of "topology," which is a representation of your service map: how the services and resources in your infrastructure relate to one another.
```

Comment:
@homelessbirds what are we doing about the titles inside the docs? There's a link to an applied intelligence doc.